### PR TITLE
New facebook auth for firebase

### PR DESCRIPTION
**Pull request by MOPineyro**
Facebook auth has changed for firebase: https://www.firebase.com/docs/ios/guide/login/facebook.html

Just a quick change to firebase_facebook_helper to get it working again.

### CODE REVIEW

File: lib/firebase/firebase_facebook_helper.rb [hunk 1]
```diff
@@ -7,16 +7,17 @@ def self.open_facebook_session(options={}, &block)
...
-    permissions = options[:permissions] || ['public_profile']
-    allow_ui = options.fetch(:allow_login_ui, true)
-    FBSession.openActiveSessionWithReadPermissions(permissions, allowLoginUI: allow_ui,
-      completionHandler: -> (session, state, error) do
-        if error
-          block.call(error, nil)
-        elsif state == FBSessionStateOpen
-          token = session.accessTokenData.accessToken
+    fb_login = FBSDKLoginManager.alloc.init
+    fb_login.logInWithReadPermissions(["email"],
```
Comment (colinta) [inline lib/firebase/firebase_facebook_helper.rb]: Is there something wrong with using the `:permissions` option, instead of hard coding it?

**Comment by colinta** [comment]
Thanks for updating this!

**Comment by MOPineyro** [comment]
Just updated to use permissions instead of hardcoding it.

**Comment by colinta** [comment]
Nice!  I'll push a new version.

**Pull request merged**